monitoring/opentelemetry/src/main/resources/application.properties

```diff
@@ -9,4 +9,6 @@ quarkus.grpc.clients.pong.host=localhost
 quarkus.grpc.clients.pong.port=${quarkus.http.port}
 quarkus.grpc.server.use-separate-server=false
 
-quarkus.application.name=pingpong
\ No newline at end of file
+quarkus.application.name=pingpong
+quarkus.otel.traces.enabled=true
```

Comment:
Are we need to add these two properties? Both of them are set with this values by default
See https://quarkus.io/guides/all-config#quarkus-core_quarkus-log-level and https://quarkus.io/guides/opentelemetry#disable-all-or-parts-of-the-opentelemetry-extension

Reply:
When I looking at the changes the `quarkus.otel.traces.enabled=false` is not set anywhere. So why we need it to changes it?

Looking at the tests in `checkThereIsNoTreaces` you expect no traces so `quarkus.otel.traces.enabled=false` should be set at the start.
In `checkTracesAfterRestart` you modify this property to `true`, restart  and check if traces are enabled.

I hope I get it corretly.
There was problem with starting the quarkus with `quarkus.otel.traces.enabled=false`. Looking at it you use only `/hello` endpoint so maybe the solution would be start dev mode only with this class something like `@DevModeQuarkusApplication(classes = { PongResource.class })` (don't see any usage but looking at the source code it should work)

Reply:
Yes, you are correct, I will try with that annotation, and let you know.

Reply:
Well, now I've done some tweaks and changes, but then the next error comes up: 
`Failed to start quarkus [Error Occurred After Shutdown]` , I got it after restar the quarkus app and failed to start quarkus again because unsatisfied dependencies for types required from an application class not used in my tests (MicroProfileTelemetryDIResource)...
```
## Running test DevModeOpenTelemetryIT.checkThereIsNoTracesAfterRestart()
08:40:22,126 ERROR The LogManager accessed before the "java.util.logging.manager" system property was set to "org.jboss.logmanager.LogManager". Results may be unexpected.
08:40:23,621 INFO  [otel] 08:40:22,253 File change detected: /home/jcarranz/cloned_projects/quarkus-test-suite/monitoring/opentelemetry/target/DevModeOpenTelemetryIT/otel/src/main/resources/application.properties
08:40:23,621 INFO  [otel] 08:40:22,254 Restarting quarkus due to changes in application.properties.
08:40:23,622 INFO  [otel] 08:40:22,262 pingpong stopped in 0.007s
08:40:23,622 INFO  [otel] 08:40:22,323 The quarkus-resteasy-mutiny extension is deprecated. Switch to Quarkus REST instead.
08:40:23,622 INFO  [otel] This extension adds support for Uni and Multi to RESTEasy Classic, without using the reactive execution model, as RESTEasy Classic does not use it. To properly integrate Mutiny and RESTEasy, use Quarkus REST. See https://quarkus.io/guides/getting-started-reactive for detailed instructions
08:40:23,622 INFO  [otel] 08:40:22,443 Failed to start quarkus [Error Occurred After Shutdown]: java.lang.RuntimeException: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
08:40:23,622 INFO  [otel] 	[error]: Build step io.quarkus.arc.deployment.ArcProcessor#validate threw an exception: jakarta.enterprise.inject.spi.DeploymentException: Found 3 deployment problems: 
08:40:23,623 INFO  [otel] [1] Unsatisfied dependency for type io.opentelemetry.api.baggage.Baggage and qualifiers [@Default]
08:40:23,623 INFO  [otel] 	- injection target: io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource#baggage
08:40:23,623 INFO  [otel] 	- declared on CLASS bean [types=[io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource, java.lang.Object], qualifiers=[@Default, @Any], target=io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource]
08:40:23,623 INFO  [otel] [2] Unsatisfied dependency for type io.opentelemetry.api.trace.Span and qualifiers [@Default]
08:40:23,623 INFO  [otel] 	- injection target: io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource#span
08:40:23,623 INFO  [otel] 	- declared on CLASS bean [types=[io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource, java.lang.Object], qualifiers=[@Default, @Any], target=io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource]
08:40:23,624 INFO  [otel] [3] Unsatisfied dependency for type io.opentelemetry.api.trace.Tracer and qualifiers [@Default]
08:40:23,624 INFO  [otel] 	- injection target: io.quarkus.ts.opentelemetry.MicroProfileTelemetryDIResource#tracer
```

Reply:
So I think after these tries we can only check that traces are working in dev mode. What I think is that injection of variables [here](https://github.com/quarkus-qe/quarkus-test-suite/blob/main/monitoring/opentelemetry/src/main/java/io/quarkus/ts/opentelemetry/MicroProfileTelemetryDIResource.java) must have enabled traces and you can't run devmode Quarkus with specific classes. So you can either disable this one test (with link to some issue why it's not possible) or delete the test. 

This is my conclusion from what I see and what I tried.
Or do you have any other ideas @jcarranzan

Reply:
Yes, that's my idea: 
1. Create an issue related to this inconvenience with `@DevModeQuarkusApplication`.
2. Disable this test class until the issue is resolved.

Reply:
I created:
https://github.com/quarkus-qe/quarkus-test-framework/issues/1100
It was resolved 
https://github.com/quarkus-qe/quarkus-test-framework/pull/1101
And the quarkus-test-framework was upgrade with the release version with the fix, I tested locally and the tests passed OK